Bio/SeqAnalysis/SeqAnalyzer.py

```diff
@@ -68,12 +76,11 @@ def median_standard_deviation_average_length(id_seq_dictionary: dict) -> tuple:
         :Example:
         >>> SeqAnalyzer.median_standard_deviation_average_length({'protein1': 'WKQTNSLEGKQ', 'protein2': 'WKQQTNSLEGKQ'})
         0.5, 11.5, 11.5
-
         """
         list_of_lengths = [len(seq) for seq in id_seq_dictionary.values()]
-        standard_deviation = float(np.std(list_of_lengths))
-        average_length = float(np.average(list_of_lengths))
-        median_length = float(np.median(list_of_lengths))
+        standard_deviation = round(float(np.std(list_of_lengths)), 3)
+        average_length = round(float(np.average(list_of_lengths)), 3)
+        median_length = round(float(np.median(list_of_lengths)), 3)
```

Comment:
Make rounding configurable, i.e. add field in `SeqAnalyzer` class: `self.rounding = 3`